src/main.rs

```diff
@@ -164,7 +167,7 @@ fn main() {
     let (mut state, mut first_run) = state::load().map_or_else(
         |error| {
             // We couldn't load any state from disk. Log the error.
-            debug!(
+            trace!(
```

Comment:
Instead of downgrading this to `trace`, can we actually upgrade it to `warning` (or at least `info`)? This message should only appear once at the beginning of the logs, so it's not noisy at all. It could potentially indicate a serious issue affecting state storage.